blog/BCHN-lead-maintainer-report-2020-06-15.md

```diff
@@ -0,0 +1,350 @@
+
+As mentioned in our proposal from April, for technical reasons we (BCHN) are committed to the next network upgrade in November, but open to reconsideration after that.
+
+The November commitment is related to the "Automatic Replay Protection" in the upgrade specifications -- a full node version-deprecation mechanism which is sometimes referred to as the "poison pill".
+
+It makes non-upgraded nodes alter their transaction signatures so that they can no longer talk to the upgraded network once the next upgrade comes around. Effectively hard-forking non-upgraded nodes to a chain of their own. This mechanism locks us in the 6-monthly cycle, because deployed nodes are already programmed to make certain changes on that future date.
+
+Come November, BCHN plans to permanently remove this poison pill code from its major upgrade release.
+
+And after that the ecosystem will have an opportunity to re-shape the way Bitcoin Cash does network upgrades.
+
+The process is currently unhealthy. Not only are working procedures, specification time frames and the completeness of functional specifications no longer adhered to - this has contributed to developmental centralization and formation of a renewed "reference client" mindset which would stifle innovation and progress if left uncorrected.
+
+If we can collectively re-orient our ecosystem culture to allow room for technical disagreement while resolving it through evidence-based evaluation, combined with safe activation mechanisms that do not put the chain at risk of splitting by hostile hashpower, then we can reach much better decisions and a swifter advancement on the Bitcoin Cash roadmap towards global adoption.
+
+### [Project uptake](#project-uptake)
+Since IFP is no longer an immediate threat, miner signaling for BCHN has decreased.
+
+This was expected to some degree: when there is not a burning issue, miners signal less willingly for particular clients that seek to address the issue.
+
+The IFP hot button issue has passed and BCH has been safely upgraded without a chain split. Subsequently, the percentage of miners signaling for us has gone down from an all time high of almost 17% to a steady state comprising our most steadfast supporters.
+
+We are thankful to those miners who have supported us during the run-up to May and helped to keep Bitcoin Cash free of a forced levy.
+
+Our special thanks to those who continue to signal support for our software using `/BCHN/` in their coinbase strings. Thanks [bitcoin.com](https://www.bitcoin.com/) , Hashpipe and P2Pool.
+
+<figure class="text-center">
+    <img src="https://rcimg.net/images/a85cfb07/3bneA5i1D0usLADh1f7AAIro3asfIlIK1AUSOFhU.png" class="rounded img-fluid" alt="" title="">

```

Comment:
Please do not use readcash for image hosting.

You already import the images into the ` app/static/img/newsroom/2020-06-15/` folder, please make use of them. I will mark other instances of this review in this file by comments `Use locally hosted image`.